src/Encoding/FilteredStream.php

```diff
@@ -24,16 +24,22 @@
 
     /**
      * @var resource
+     *
+     * @deprecated since version 1.5, will be removed in 2.0
      */
     protected $readFilter;
 
     /**
      * @var callable
+     *
+     * @deprecated since version 1.5, will be removed in 2.0
      */
     protected $writeFilterCallback;
 
     /**
      * @var resource
+     *
+     * @deprecated since version 1.5, will be removed in 2.0
      */
     protected $writeFilter;
```

Comment:
I think this can be safely removed as it was always null and never used, even if it's BC i assume nobody use this (otherwise i really don't understand why xD)

@dbu @sagikazarmark ok to remove this even if it's somehow BC (never used, always null value in a protected scope so ...) ?